README.md

```diff
@@ -39,6 +39,7 @@
 ----
 
 :fire: *News* :fire:
+- [Aug 2025] **OpenAI GPT-OSS models** (gpt-oss-120b, gpt-oss-20b) LoRA/Full Finetuning with 🤗 Accelerate on any cloud: [**example**](./llm/gpt-oss-sft/)
```

Comment:
How about combining this with the serving news below : )

Reply:
Updated